src/WordPress/Database/MysqlDatabase.php

```diff
@@ -330,14 +330,8 @@ public function import(string $dumpFilePath): int
     public function dump(string $dumpFile): void
     {
         try {
-            $dump = new class($this->dsn, $this->dbUser, $this->dbPassword) extends Mysqldump {
-                public function start($filename = '')
-                {
-                    $this->dumpSettings['add-drop-table'] = true;
-                    $this->dumpSettings['add-drop-database'] = true;
-                    return parent::start($filename);
-                }
-            };
+            $dumpSettings = ['add-drop-table'=> true, 'add-drop-database' => true];
```

Comment:
will the composer-35.json file use the same source code? if so, this line here might need a if-class-exist or similar construct so it works with both Mysqldump implementations?

Reply:
Yes, v3.5 is transpiled from v4 and will use the same code.
I've chased a missing db version update issue that was failing the tests, when that is solved, I will test v3.5 for issues, and will, likely put in place the conditional code.